File: lab_00pre/tester/gen.js
// const http = require('http')
const prompt = require('prompt-sync')()

let n = prompt("n : ")
let range = prompt("range : ")
let res = ""

for (let i = 0; i < (n * 2); i++) {
    res += `${Math.floor(Math.random() * range)}`
    if (i != (n * 2) - 1)
        res += " "
}

console.clear()
console.log(`n : ${n}\n${res}`);

File: lab_00pre/gen.py
import math, random


n = int(input("n : "));
r = int(input("max weight : "))
res = ""

for i in range(n * 2):
    res = res + str(math.floor(random.random() * r) + 1)
    if i != (n * 2) - 1:
        res = res + " "

# for i in range(n * 2):
#     res = res + str(math.floor(random.random() * r) + 1)
#     if i != (n * 2) - 1:
#         res = res + ", "

print(res)
